lib/java/gradle/unitTests.gradle

```diff
@@ -68,6 +68,20 @@ test {
     include '**/Test*.class'
     exclude '**/Test*\$*.class'
 
+    println 'Check if msvc.profile exists'
```

Comment:
It seems to me this fix is in the wrong place.  I don't like having the C++ compiler type passed down into the java build so it can disable a test.  The compiler code be modified somehow.

Reply:
I did not want to disable the entire `JavaTest` just for a single test suite.